mantl/mantl-dns/spec.yml

```diff
@@ -0,0 +1,56 @@
+---
+name: mantl-dns
+version: 1.0.0
+license: ASL 2.0
+iteration: 1
+vendor: Asteris
+url: https://github.com/asteris-llc/mantl-packaging
+description: DNS integration with Consul
+type: rpm
+
+depends:
+  - dnsmasq = 2.66
```

Comment:
since this is something that has been broken by upgrades in the past, I figured we should be more cautious with the version pinning here.